src/libstd/path.rs

```diff
@@ -1324,11 +1368,14 @@ impl cmp::Ord for Path {
 }
 
 /// Freely convertible to a `Path`.
+#[stable(feature = "rust1", since = "1.0.0")]
 pub trait AsPath {
```

Comment:
Should this hold off on being stable while generic conversion traits are landing?